### PR TITLE
Fix mametesters bug #3041

### PR DESCRIPTION
**Pull request by palazzol**
Update clock frequencies based on video evidence and PCB photos
Updated thrust control from toggle to momentary per actual game

### CODE REVIEW

**Comment by happppp** [comment]
The fast/slow lever is spring-loaded? If so, could you add a comment to IPT_BUTTON3? So noone in the future changes it back to IPT_TOGGLE (MAME uses that for stick shift levers like in Outrun)

**Comment by palazzol** [comment]
Good idea - yes the level goes back to SLOW unless you hold it down, despite the way it looks.  I'll add a comment to that effect

File: src/mame/drivers/starshp1.cpp [hunk 1]
```diff
@@ -197,7 +197,10 @@ static INPUT_PORTS_START( starshp1 )
 	PORT_DIPNAME( 0x20, 0x20, "Extended Play" )
 	PORT_DIPSETTING(    0x00, DEF_STR( No ) )
 	PORT_DIPSETTING(    0x20, DEF_STR( Yes ) )
-	PORT_BIT( 0x40, IP_ACTIVE_LOW, IPT_BUTTON3 ) PORT_TOGGLE
+	/* IPT_BUTTON3 is the Speed lever (Throttle)                */
+	/* This is _not_ IPT_TOGGLE, even though it looks like one. */
+	/* It returns to SLOW unless you hold it down (FAST)        */
+	PORT_BIT( 0x40, IP_ACTIVE_LOW, IPT_BUTTON3 )
```
Comment (cuavas) [inline src/mame/drivers/starshp1.cpp]: Can you please either use C++ line comments, or a single C block comment?  Doing it like this gets the worst of both worlds - hard to reformat, visual clutter at the beginning and end of each line, and stuff that needs to be realigned if lines change length.

**Comment by palazzol** [comment]
OK - I went with C++ style comments.  Thanks for your review

**Pull request merged**